### PR TITLE
Story - IO-167 - Hankekortti connect to local API

### PR DESCRIPTION
**Pull request by EngbergJonas**
- axios
- axios-interceptor, 
- projectCard service
- projectCard thunk for calling the service and updating the redux state
- Dotenv to webpack to handle .env files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,8 +11,15 @@
     - [State and storage](#state-and-storage)
     - [Routing](#routing)
     - [Testing](#testing)
+    - [Localization](#localization)
     - [Collection of scripts](#collection-of-scripts)
 
+### Setup
+
+1. `npm run create-env` to create your own development and production .env files
```
Comment (kgunga) [inline README.md:19]: use yarn create-env instead

File: README.md [hunk 2]
```diff
@@ -10,8 +11,15 @@
...
+2. `yarn install`
+3. `npm start`
```
Comment (kgunga) [inline README.md:21]: use yarn start instead. we should use yarn tool everywhere

**Review by kgunga — CHANGES_REQUESTED** [review]

**Review by kgunga — CHANGES_REQUESTED** [review]
I think we should use react-scripts instead of using directly webpack. react-scripts already wraps webpack.

File: README.md (hunk 1 above)
Comment (EngbergJonas) [reply README.md]: Resolved.

File: README.md (hunk 2 above)
Comment (EngbergJonas) [reply README.md]: Resolved.

**Review by kgunga — APPROVED** [review]
Looks good

**Pull request merged**